raft/src/main/java/org/apache/kafka/raft/internals/AddVoterHandler.java.orig

```diff
@@ -0,0 +1,379 @@
+/*
```

Comment:
I do not think you intended to submit this `.orig` file.

Reply:
Removed.